src/modules/netinstall/README.md

```diff
@@ -0,0 +1,28 @@
+# Netinstall module
+
+The netinstall module allows distribution maintainers to ship minimal ISOs with only a basic set of preinstall packages. At installation time, the user is presented with the choice to install groups of packages from a predefined list.
+
+Calamares will then invoke the correct backend to install the packages.
+
+## How are packages configured?
+The *netinstall.conf* file should have this format:
+
+    ----
+    groupsUrl: <URL to YAML file>
```

Comment:
Any reason why the YAML file must be remote? Could it be a local YAML file, or even a YAML object included in netinstall.conf?